### PR TITLE
Created main components of product view.

### PR DESCRIPTION
**Pull request by vuamy**
Notes:
I assumed the buttons at the bottom were to switch through the images, but this could be changed. I also tried to not add width/height for static images but it was giving me issues, so I included it for the icons. Also, the media display could be changed since the slider for images is all the way at the bottom.

### CODE REVIEW

File: app/(pages)/products/product-view/_components/ProductView.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    description:
+      'Made from ultra-soft jersey cotton, it gets softer with every wash. Durable double stitching and ribbed crewneck maintains its shape wash after wash.',
+    details:
+      'Made from ultra-soft jersey cotton, it gets softer with every wash. Durable double stitching and ribbed crewneck maintains its shape wash after wash.',
+  };
+
+  const [currentImage, setCurrentImage] = useState(1);
+
+  function addCurrentImage() {
+    if (currentImage === displayImages.length) {
```
Comment (Austin2Shih) [inline app/(pages)/products/product-view/_components/ProductView.tsx:21]: you could use the modulo operator in the future, but this works fine

File: app/(pages)/products/product-view/_components/ProductView.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+              <p className={styles.info_label}>Details:</p>
+              <p>{fakeItem.details}</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className={styles.slider}>
+        <button onClick={subCurrentImage}>
+          <Image
+            src="/icons/chevron-left.svg"
```
Comment (Austin2Shih) [inline app/(pages)/products/product-view/_components/ProductView.tsx:116]: For any image that you expect to be pulled locally (icons, static images) you should use static image rendering

https://nextjs.org/docs/app/building-your-application/optimizing/images#local-images

This only applies to the icons on this page tho, the shirt images are expected to come externally so those are fine.

**Review by Austin2Shih — CHANGES_REQUESTED** [review]
Looks good! Just a few changes that I requested

File: app/(pages)/products/product-view/_components/ProductView.tsx (hunk 2 above)
Comment (vuamy) [reply app/(pages)/products/product-view/_components/ProductView.tsx]: Added! (I think)

File: app/(pages)/products/product-view/_components/ProductView.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
+'use client';
+import styles from './ProductView.module.scss';
+import Image from 'next/image';
+import listedItem from '../../../../../public/icons/check_circle_fill.svg';
```
Comment (Austin2Shih) [inline app/(pages)/products/product-view/_components/ProductView.tsx:4]: This should just be imported from ```/public/icons/...```

**Review by Austin2Shih — APPROVED** [review]
Looks good to me!

**Pull request merged**